ginga/rv/plugins/Preferences.py

```diff
@@ -268,6 +268,40 @@
 images are being loaded into a channel frequently (e.g., a low frequency
 video feed), it may be undesirable to create thumbnails for all of them.
 
+**General Preferences**
+
+The "Num Images" setting specifies how many images can be retained in
+buffers in this channel before being ejected.  A value of zero (0) means
+unlimited--images will never be ejected.  If an image was loaded from
+some accessible storage and it is ejected, it will automatically be
+reloaded if the image is revisited by navigating the channel.  A setting
+of 1 to 4 is typical.
+
+The "Sort Order" setting determines whether images are sorted in the
+channel alphabetically by name or by the time when they were loaded.
```

Comment:
Perhaps also note that the order they are loaded is random due to multi-threading (it is, right?). Some users have asked about how things are loaded in Ginga.

Reply:
Yes, if you load several images simultaneously and the sort order is set to load time then the order can appear random due to the use of threading.